src/state.rs

```diff
@@ -418,12 +439,9 @@ mod test {
         })).is_ok());
 
         let accept_thread = thread::spawn(move || {
-            // Currently acceptor starts accepting implicitly,
-            // once fixed (https://github.com/maidsafe/crust/issues/316),
-            // the following lines should be uncommented.
-            //for a in acceptors {
-            //    assert!(::transport::accept(&a).is_ok());
-            //}
+            for a in acceptors {
+                assert!(State::accept(&a).is_ok())
+            }
```

Comment:
I'm not sure this has changed, the acceptor still seems to start accepting implicitly on creation:

https://github.com/maidsafe/crust/blob/d9aa489cc4ef84d0557c983c50650334cbd52141/src/transport.rs#L308
https://github.com/maidsafe/crust/blob/d9aa489cc4ef84d0557c983c50650334cbd52141/src/tcp_connections.rs#L40